Add participant

maticzav opened the pull request:
#1 referenced!

MJ10 commented on PARTICIPANTS/maticzav.txt:
```diff
@@ -0,0 +1,3 @@
+Name: Matic Zavadlal
```
The name of the directory should be `Participants`

salman-bhai commented:
Are there any updates or should we close this PR as `spam`?

maticzav commented:
Hi! No, I shall fix this today. Hope that you can wait :smile:

maticzav commented:
Fixed changes in #467 as I was unable to change directory name from `PARTICIPANTS` to `Participants` in local file system. Hope I fixed everything and thank you so much that you waited.
